x-pack/auditbeat/module/system/packages/packages.go

```diff
@@ -148,7 +148,7 @@ func (ms *MetricSet) Fetch(report mb.ReporterV2) {
 
 		for _, pkgInfo := range installed {
 			pkgInfoMapStr := pkgInfo.(*Package).toMapStr()
-			pkgInfoMapStr.Put("status", "new")
+			pkgInfoMapStr.Put("status", "NEW")
```

Comment:
Seems like these changes are not relevant to the socket metricset.

Reply:
Agreed. Changes to this file as well as `processes.go`

Reply:
Moved them out. Also, what used to be `status` is now `event.action`.